style.css

```diff
@@ -0,0 +1,415 @@
+body {
+  min-width: 375px;
+  margin: 0;
+  padding: 0;
+  font-family: Montserrat, Arial, sans-serif;
+}
+
+.header {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+  justify-content: space-between;
+  min-height: 70px;
+    -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.header__lang {
+  display: none;
+}
+
+.header__logo {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  text-decoration: none;
+  position: absolute;
+  width: 98px;
+  height: 37px;
+  margin-left: 29px;
+  margin-top: 22px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 37px;
+  text-align: center;
+  letter-spacing: 0.07em;
+  color: #fff;
+}
+
+.header__menu {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-item-align: end;
+  align-self: flex-end;
+  width: 70px;
+  height: 70px;
+  background: #f5f5f3;
+  cursor: pointer;
+}
+
+.header__burger {
+  -ms-flex-item-align: center;
+  -ms-grid-row-align: center;
+  align-self: center;
+  margin: auto;
+}
+
+.header__burger-line {
+  display: block;
+  width: 26px;
+  height: 3px;
+  background: #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5676737)